### DISCUSSION TITLE
RLadies theme

### DISCUSSION THREAD

**Discussion by PythonCoderUnicorn** [quarto-dev/quarto-cli#6261] [Show and Tell]
Hello Quarto !

I have read through the docs on presentation slides and now know about them, including theming in Sass.

I made a Sass theme for RLadies group to be added to be a future option for users. 

I first tried just having the [Sass variables](https://quarto.org/docs/presentations/revealjs/themes.html#sass-variables) as shown in the table, but had limited success having it work beyond basics in the `body{}` section after declaring all the variables. So I ended up copying the code from `darkly.scss` in order for my Quarto presentation to render, otherwise I would get an error about my Sass `code not containing at least one layer boundary`.  My RLadies theme runs in RStudio with no issues at all with animations or anything. 

The only changes for an **RLadies theme** from the **darkly** theme is:

```
# changed the purple hex values and added a purple2
$purple:  #562457 !default;
$purple2: #b77ef0 !default;
```

for the body
```
// Body

$body-bg:   $purple !default;  // <--- changed
$body-color:     $white !default;

@function body-mix($weight) {
    @return mix($body-bg, $body-color, $weight);
}
```

and for the links
```
// Links

$link-color:   $purple2 !default; // <--- changed
```

Please add this theme to your main themes.  Thank you.

**Comment by jjallaire** [comment]
Here's another R Ladies Quarto Reveal theme: https://github.com/beatrizmilz/quarto-rladies-theme

We are soon going to be making available a "custom formats" capability that will make it extremely easy to create a custom them and distribute it from a URL. Will post back here when that is ready!

**Comment by PythonCoderUnicorn** [comment]
@jjallaire  thanks !

My version is for the dark theme of RLadies and not the same as above linked light version.

**Comment by jjallaire** [comment]
Okay, thanks for the clarification! I think once we have the custom formats it would be possible to merge them and provide a YAML option for dark vs. light.

**Comment by PythonCoderUnicorn** [comment]
I uploaded my code into a repo 

here is my repo https://github.com/PythonCoderUnicorn/Quarto-RLadies
and here is the slides https://pythoncoderunicorn.github.io/Quarto-RLadies/
